roc/convert_test.go

```diff
@@ -0,0 +1,56 @@
+package roc
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_go2cBool(t *testing.T) {
+	tests := []struct {
+		arg    bool
+		result uint
+	}{
+		{true, 1},
+		{false, 0},
+	}
+	for _, tt := range tests {
+		t.Run(strconv.FormatBool(tt.arg), func(t *testing.T) {
+			assert.Equal(t, tt.result, uint(go2cBool(tt.arg)))
+		})
+	}
+}
+
+func Test_go2cStr(t *testing.T) {
+	tests := []struct {
+		name   string
+		arg    string
+		result []char
+	}{
+		{name: "str", arg: "str", result: []char{'s', 't', 'r', '\x00'}},
+		{name: "str00str", arg: "str\x00", result: []char{'s', 't', 'r', '\x00', '\x00'}},
+		{name: "empty", arg: "", result: []char{'\x00'}},
```

Comment:
* let's rename `name: "str00str"` to `name: "str0str"` because there is only one zero byte
* let's add a test when zero byte is in the end

Reply:
done